app/code/Magento/Cookie/Test/Mftf/Test/StorefrontVerifyUnsecureCookieTest.xml

```diff
@@ -21,16 +21,28 @@
             <group value="configuration"/>
         </annotations>
         <before>
+            <!-- Go to storefront home page -->
+            <actionGroup ref="StorefrontOpenHomePageActionGroup" stepKey="openStoreFrontHomePage"/>
+            <executeJS function="return window.location.host" stepKey="hostname"/>
+            <!-- Login as admin -->
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+            <magentoCLI command="config:set web/unsecure/base_url https://{$hostname}/" stepKey="setUnsecureBaseURL"/>
```

Comment:
Please pay attention to the description of the test case. it says "Verify that cookie are not secure on storefront over **http**". Please add another test if you want to cover different scenario. As I see you probably want to test the scenario when https is configured but use_in_frontend is set 0.